src/main/java/com/massivedisaster/bintraydeployautomator/utils/GradleUtils.java

```diff
@@ -15,12 +20,34 @@
      * @param tasks            gradle tasks to execute.
      * @param arguments        arguments.
      */
-    public static void runGradle(@NotNull ProjectConnection gradleConnection, @NotNull String[] tasks, @NotNull String... arguments) {
-        gradleConnection.newBuild()
-                .forTasks(tasks)
-                .withArguments(arguments)
-                .setStandardOutput(System.out)
-                .setStandardError(System.err)
-                .run();
+    public static void runGradle(@NotNull ProjectConnection gradleConnection, @NotNull String output, @NotNull String[] tasks, @NotNull String... arguments) {
+        try {
+            OutputStream os;
```

Comment:
This file creation stuff should be in a separated method.

Reply:
Fixed in next commit.